klippy/extras/display/uc1701.py

```diff
@@ -164,6 +168,7 @@ class UC1701(DisplayBase):
     def __init__(self, config):
         io = SPI4wire(config, "a0_pin")
         DisplayBase.__init__(self, io)
+        self.x_offset = config.getint('x_offset', 0)
```

Comment:
The UC1701 doesn't (and can't) support an x_offset.  This should be added to `class SH1106` as it's only possible to have an x_offset if there are 132 columns.  Also, the max/min should be checked: `x_offset = config.getint('x_offset', 0, minval=0, maxval=3)`